### PR TITLE
Dev radosavantonio - welcome, connect and skills

### PR DESCRIPTION
**Pull request by RadosavAntonio**

### CODE REVIEW

File: web/css/style.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+.description {
+  margin: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 30px;
+  color: #808080;
+}
+
+.bAtention {
```
Comment (marianzburlea) [inline web/css/style.css]: why did you use camel case?

File: web/css/style.css [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  color: #fff;
+  border-radius: 50%;
+  height: 40px;
+  width: 40px;
+  border: 0;
+}
+
+
+
+.tCentered {
```
Comment (marianzburlea) [inline web/css/style.css]: replace by `some-name`

File: web/css/style.css [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  width: 100%;
+  height: 100%;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 8vw;
+}
+
+.bDisplay__onahau {
```
Comment (marianzburlea) [inline web/css/style.css]: have fun with updating the name of the selectors as well as the html attribute values

File: web/css/style.css [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  background-color:#fafafa;
+}
+
+.skills-legend_item > span[class^="my-cv-"] {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  height: 30px;
+  /* font-size: 6vw; */
```
Comment (marianzburlea) [inline web/css/style.css]: once you're happy with your code, remove unnecessary comments

File: web/css/style.css [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+  padding-right: 20px;
+}
+
+.legend-description {
+  padding: 0 10px;
+  flex-grow: 1;
+  display: inline-flex;
+  justify-content: left;
+  align-items: center;
+}
```
Comment (marianzburlea) [inline web/css/style.css]: write a new line

File: web/index.html [hunk 6]
```diff
@@ -1,11 +1,183 @@
...
+    
+    <grid class="connect">
+      <c span="row" height="30"></c>
+      <c span="row">
+        <h3 class="section-title">Let's connect</h3>
+      </c>
+      <c span="row"></c>
+      <c span="row">
+        <div class="image-wrapper">
+          <img class="avatar" src="image/antonio.jpeg" alt="">
```
Comment (marianzburlea) [inline web/index.html]: you're missing the value of the alt attribute

File: web/css/style.css (hunk 1 above)
Comment (RadosavAntonio) [reply web/css/style.css]: Done!
Now I understand camel case are for JS and not for CSS

File: web/css/style.css (hunk 2 above)
Comment (RadosavAntonio) [reply web/css/style.css]: done

File: web/css/style.css (hunk 3 above)
Comment (RadosavAntonio) [reply web/css/style.css]: done

File: web/css/style.css (hunk 4 above)
Comment (RadosavAntonio) [reply web/css/style.css]: done

File: web/css/style.css (hunk 5 above)
Comment (RadosavAntonio) [reply web/css/style.css]: done

File: web/index.html (hunk 6 above)
Comment (RadosavAntonio) [reply web/index.html]: done

File: web/index.html [hunk 7]
```diff
@@ -42,74 +42,77 @@ <h3 class="section-title">About me</h3>
...
+              <i class="my-cv-instagram"></i>
+            </a>
+          </c>
+          <c span="1">
+            <a class="button-display__janna">
+              <i class="my-cv-twitter"></i>
+            </a>
+          </c>
+          <c span="1">
+            <a class="button-display__doublePearlLusta">
```
Comment (marianzburlea) [inline web/index.html]: `button-display__doublePearlLusta`
`button-display__double-pearl-lusta`

**Pull request merged**